dali/test/python/test_operator_select.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from numpy import random
+from numpy.core.fromnumeric import shape
+from nvidia.dali import Pipeline
```

Comment:
Why not use decorator?

Reply:
The decorator makes it easier to have a function that returns a pipeline (it does away with forwarding arguments, `with pipe`, `set_outputs`, etc - here there's no such function and forwarding arguments would likely make the code _more_ repetitive.